tests/rados/test_bz_2213766.py

```diff
@@ -0,0 +1,138 @@
+from ceph.ceph_admin import CephAdmin
```

Comment:
Please add docstrings for the test file.

Reply:
Added doc string

https://github.com/red-hat-storage/cephci/blob/262205193a7497c6e25014f51a479518e7a1ba85/tests/rados/test_bz_2213766.py#L1-L6